docker/docker_client.go

```diff
@@ -533,6 +533,8 @@ func (c *dockerClient) getBearerToken(ctx context.Context, challenge challenge,
 	defer res.Body.Close()
 	switch res.StatusCode {
 	case http.StatusUnauthorized:
+		err := client.HandleErrorResponse(res)
+		logrus.Errorf("Server response when trying to obtain an access token: \n%q", err)
```

Comment:
… wait, `Errorf`? https://github.com/containers/libpod/issues/3884 talks about debug level.

It wouldn’t be the _first_ time the library calls `logrus.Errorf`, but it’s only the third, and it can be pretty surprising for callers who don’t expect any output.

I don’t feel _too_ strongly about this, if there is consensus, but I do wonder whether it was intentional or maybe just an artifact from testing.

Reply:
sorry, changed to debug level